server/views.py

```diff
@@ -155,6 +155,10 @@ def retrieve(request, request_id):
     cleanup()
     the_request = get_object_or_404(Request, pk=request_id)
     if the_request.approved == True and the_request.current==True:
+        if hasattr(settings, 'ROTATE_VIEWED_SECRETS'):
+        if settings.ROTATE_VIEWED_SECRETS:
```

Comment:
This should be indented or combined on the above line with an `and` statement, right?